src/yuzu/configuration/configure_input_player.cpp

```diff
@@ -882,6 +946,7 @@ void ConfigureInputPlayer::UpdateControllerIcon() {
     // "nonstandard" to use an image through the icon support)
     const QString stylesheet = [this] {
         switch (GetControllerTypeFromIndex(ui->comboControllerType->currentIndex())) {
+        case Settings::ControllerType::Pokeball:
         case Settings::ControllerType::ProController:
```

Comment:
minor nit, swap the 2
```suggestion
        case Settings::ControllerType::ProController:
        case Settings::ControllerType::Pokeball:
```